features/contract-state/DisputeParams.tsx

```diff
@@ -25,20 +27,38 @@ const fromWei = ethers.utils.formatUnits;
 
 const DisputeParams = () => {
   const { empState } = EmpState.useContainer();
-
   const {
     disputeBondPct,
     disputerDisputeRewardPct,
     sponsorDisputeRewardPct,
     withdrawalLiveness,
   } = empState;
+
+  const { dvmState } = DvmState.useContainer();
+  const { finalFee } = dvmState;
+  const { symbol: collSymbol } = Collateral.useContainer();
+
   const withdrawalLivenessInMinutes = (Number(withdrawalLiveness) / 60).toFixed(
     2
   );
 
   return (
     <Box>
       <Typography variant="h5">Dispute Params</Typography>
+      <Status>
+        <Label>
+          Liquidation bond (
+          <Link
+            href={DOCS_MAP.FINAL_FEE}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Docs
+          </Link>
+          ){`: `}
+        </Label>
+        {finalFee ? finalFee : "N/A"} {collSymbol}
```

Comment:
note this component does not use the `renderComponent` notation. There are a few other places through the code that don't and I didn't want to muddle this PR too much with changing this so I used the same pattern as the rest of this file.

Reply:
nits:
- This check should be `finalFee !== null`
- don't you also have to check if `collSymbol` is null?

Reply:
done. will need to apply the same cleaning process on a bunch of other places through the dapp.